compiler/src/iree/compiler/Codegen/Common/TransformDialectStrategies.cpp

```diff
@@ -238,11 +238,11 @@ static Value distributeVectors(ImplicitLocOpBuilder &b, Value funcH,
 template <typename TileSizesType>
 static Value buildReductionStrategyBlockDistributionPart(
     ImplicitLocOpBuilder &b, Value variantH, Value originalFillH,
-    Value originalGenericH, TileSizesType tileSizes0Generic) {
+    Value reductionH, Value fusionRootH, TileSizesType tileSizes0Generic) {
```

Comment:
Coming back to this in a second pass, I think I would just pass `ReductionStrategyTileSizesListType` everywhere and pull the proper subelement explicitly.

Reply:
This is literally what https://github.com/iree-org/iree/pull/11222 was doing. Also note that this PR doesn't touch tile sizes.